web/libs/datamanager/src/components/CellViews/index.js

```diff
@@ -15,3 +17,9 @@ export { StringCell as Text } from "./StringCell";
 export { VideoCell as Video } from "./VideoCell";
 export { ProjectCell as Project } from './ProjectCell';
 
+export function normalizeCellAlias(alias) {
+  const safeAlias = alias
```

Comment:
```suggestion
  // remove trailing separators to make `toStudlyCaps` safe
  const safeAlias = alias.replace(/[-_\s]+$/g, '');
```

I believe that we could not break this line, since this line is not more than 50 chars, I don't think we should break it; doing so makes reading easier

Reply:
great observation @juliosgarbi +1 to this